packages/react/package.json

```diff
@@ -24,10 +24,17 @@
     "classnames": "^2.3.1",
     "qrcode": "^1.4.4"
   },
+  "workspaces": {
+    "nohoist": [
+      "@types/testing-library__jest-dom",
+      "@types/testing-library__jest-dom/**"
+    ]
+  },
```

Comment:
Normally any `workspaces` declarations go on the root `package.json`.

I noticed these warnings are occurring on `yarn install` now:

> **warning** nohoist config is ignored in "@aws-amplify/ui-react" because it is not a private package. If you think nohoist should be allowed in public packages, please submit an issue for your use case.

So this doesn't seem to be doing anything.  Can you test removing this change?

Reply:
I tested that. It looks like the type files cannot be loaded correctly in Jest env when they are on root.

Reply:
So yarn _is_ using this `nohoist` config despite these warnings saying `config is ignored in` on install?

```
warning nohoist config is ignored in "@aws-amplify/ui-react" because it is not a private package. If you think nohoist should be allowed in public packages, please submit an issue for your use case.
warning nohoist config is ignored in "@aws-amplify/ui-react" because it is not a private package. If you think nohoist should be allowed in public packages, please submit an issue for your use case.
warning nohoist config is ignored in "@aws-amplify/ui-react" because it is not a private package. If you think nohoist should be allowed in public packages, please submit an issue for your use case.
```

Reply:
Yup, I think so. Honestly, I don't like this `no hoist` config but if we removed that, the reality is the custom jest matchers from `'@testing-library/jest-dom'` cannot get recognized in runtime though we can get type reference.